ui_testing/testcases/basic_tests/test05_contacts.py

```diff
@@ -741,3 +741,102 @@ def test030_filter_by_contact_location(self):
             row_data = self.base_selenium.get_row_cells_dict_related_to_header(row=table_records[counter])
             self.assertEqual(row_data['Location'], data_to_filter_with)
             counter = counter + 1
+
+    
+    def test031_add_contact_title(self):
+        """
+        LIMS-6491
+        Contacts: Contact person will have a title field to choose (Mr. or Ms)
+        """
+        # create contact with Mr.
+        contact_data = self.contact_page.create_update_contact(save=False, contact_persons=False)
```

Comment:
if save = False , contact won't be saved

Reply:
It will save contact even if Save = False